aggregator/src/aggregator/http_handlers.rs

```diff
@@ -266,6 +266,34 @@ where
     }
 }
 
+/// A Trillium handler that returns an empty body.
+#[derive(Clone, Copy)]
+struct EmptyBody {
+    media_type: &'static str,
+    retry_after: &'static str, // TODO: Make this editable? TKTK
+    status: Status,            // TODO: Make this editable? TKTK
+}
+
+impl EmptyBody {
+    fn new(media_type: &'static str) -> Self {
+        Self {
+            media_type,
+            retry_after: "2",
+            status: Status::Ok,
+        }
+    }
+}
+
+#[async_trait]
+impl Handler for EmptyBody {
+    async fn run(&self, conn: Conn) -> Conn {
+        conn.with_response_header(KnownHeaderName::ContentType, self.media_type)
```

Comment:
Content-Type for an empty body seems weird?

Reply:
It certainly does. I think I got my wires crossed in a Slack conversation earlier, as I had dropped it in the first pass, but then I thought I saw something... Anyway, yeah, I'll go strip that.